### ISSUE TITLE
Tracker list icons missing alt-text or tooltip descriptions

### ISSUE THREAD

**Issue by lenacohen** [EFForg/privacybadger#3138]
Several icons that appear in the expanded tracker list in Privacy Badger's popup are not screanreader accessible. They are missing alt-text and the tooltips that appear on hover are not read by screenreaders. A possible fix is using the tooltip language as alt-text for the icons.

The inaccessible icons are:

- The arrow to return control to Privacy Badger

<img width="319" height="98" alt="Image" src="https://github.com/user-attachments/assets/77e8ae34-8875-4921-a7f2-274c69605897" />

- The icons explaining what each slider option does

<img width="271" height="90" alt="Image" src="https://github.com/user-attachments/assets/3611ef72-4ac1-4d5b-99a4-5a12dc1adb13" />

- The DNT icon 

<img width="261" height="69" alt="Image" src="https://github.com/user-attachments/assets/5aecfeb1-8205-420d-80ab-f17022089b2e" />
